contrib/cirrus/runner.sh

```diff
@@ -338,6 +338,10 @@ function _run_gitlab() {
     return $ret
 }
 
+function _run_fcos_image_build() {
+    podman build -t quay.io/podman/fcos:podman-next -f ./contrib/podman-next/fcos-podmanimage/Containerfile .
```

Comment:
The cirrus task defines `CTXDIR` which is fine, but it's not actually referenced here.  it's a minor thing, but it would be nice to remove that definition, or actually make use of it.

Reply:
sure

Reply:
done. PTAL